### DISCUSSION TITLE
Github Copilot Chat in Mobile

### DISCUSSION THREAD

**Discussion by FranLegon** [community/community#121554] [Copilot Conversations] [answered]
![Screenshot_20240429_153233_Chrome.jpg](https://github.com/community/community/assets/6463827/88e3ec42-c2a7-47bd-b284-b508d303ccc6)

Found this, but still inaccessible. Have there been news on mobile support?

**Comment by itzzjb** [comment]
No still not. I think they will release in the near future.

**Comment by kenny-kvibe** [comment]
+1

**Comment by FranLegon** [comment, marked as answer]
It is now available https://github.blog/2024-05-07-github-copilot-chat-in-github-mobile/
